lib/rspec/core/configuration.rb

```diff
@@ -155,7 +155,7 @@ def deprecation_stream=(value)
             "it to take effect, or use the `--deprecation-out` CLI option. " \
             "(Called from #{CallerFilter.first_non_rspec_line})"
```

Comment:
I tried, but weirdness happened, I might have time to give it another go, but maybe as another follow up pr so it doesn't block us releasing 3.6